scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -74,12 +127,23 @@ def expm_multiply(A, B, start=None, stop=None, num=None, endpoint=None):
         Number of time points to use.
     endpoint : bool, optional
         If True, `stop` is the last time point.  Otherwise, it is not included.
+    traceA : scalar, optional
+        Trace of `A`. If not given the trace is estimated for linear operators,
+        or calculated exactly for sparse matrices. It is used to precondition
+        `A`, thus an approximate trace is acceptable.
+        For linear operators, `traceA` should be provided to ensure performance
+        as the estimation is not reliable.
```

Comment:
This reads a bit strange as "if it is not reliable why do we even bother". I think what you mean here is 
```suggestion
        as the estimation is not guaranteed to be reliable for all possible cases.
```

or something similar because we are generating a warning about this and should be obvious where and why it is coming from.